doc/getting-started/getting-started.rst

```diff
@@ -132,31 +135,20 @@ To set up the ACRN build environment on the development computer:
       cd ~/acrn-work
       git clone https://github.com/projectacrn/acrn-hypervisor.git
       cd acrn-hypervisor
-      git checkout v3.1
+      git checkout master
 
       cd ..
       git clone https://github.com/projectacrn/acrn-kernel.git
       cd acrn-kernel
-      git checkout acrn-v3.1
+      git checkout master
 
-#. Install Python package dependencies:
 
-   .. code-block:: bash
-
-      cd ~/acrn-work/acrn-hypervisor
-      sudo python3 -m pip install -r misc/config_tools/requirements.txt
+#. Configure git with your name and email address:
```

Comment:
Why do you need to configure git with your name and email here?  Do these changes do a commit, for example?  If not, then you don't need the Git name and email set.  We do tell the developer to set these if they're going to contribute to the project, but they shouldn't be needed in this GSG.

I notice that the debian/debian_build.sh script checks for the git user.email setting, but why?

Reply:
The git user email is used for `gpb dch` to generate debian/changelog where the email is mandatory needed, and the debian/changlog is a required file for ACRN debian packages generation.

Reply:
OK.  Thanks for explaining.  It's a new DX requirement.  It sounds like you need a name and email address for the debian/changelog but that has nothing to do with the git settings, other than it's convenient to borrow those settings to get a name and email address.  We'll need to explain this in the GSG (why we're tracking the user's name and email address and where it's being used).  Currently you can try ACRN anonymously.

Reply:
Can I update the debian/debian_build.sh script to let the `gbp dch` to get the email address for debian/changlog from one of the 3 ways with priority from high to low:  using environment variable 'EMAIL' or 'DEBEMAIL' if existed(this is actually `gbp dch` required), user.email in git config if existed(set user.email to environment variable 'EMAIL' or 'DEBEMAIL' for `gbp dch`),  or using a dummy address "acrn@projectacrn.github.io"? Then users can try ACRN anonymously without setting user.name/user.email with git config at this step.

Reply:
I like allowing anonymous trial of ACRN, so using the three methods to get an email address works for me.  I recommend we change the dummy address to a real address we have available (projectacrn@gmail.com).  Your thoughts @NanlinXie and @junjiemao1

Reply:
I am ok for this email address: [projectacrn@gmail.com](mailto:projectacrn@gmail.com)

Reply:
added dummy address to let anonymous trial of ACRN in PR: #[8308 ](https://github.com/projectacrn/acrn-hypervisor/pull/8308)